Error: Cannot find module 'tslib' / Failed to load module "appmenu-gtk-module" (Rocket.Chat 3.0.0-beta)

<!--
Thanks for opening an issue! A few things to keep in mind:

- Before reporting a bug, please try reproducing your issue with the latest
  version.
- Please verify that the bug is related to the desktop app, and NOT the main web
  app by testing in Chrome/Firefox.
- If the issue also occurs in the browser, report to
  https://github.com/RocketChat/Rocket.Chat instead
-->

## My Setup

- Operating System: Ubuntu 20.04
- App Version: 3.0.0-beta
- Installation type: <!-- exe, dmg, package manager etc. --> Snap

<!-- Answer questions by putting x in box, e.g. [x] -->
- [x] I have tested with the latest version
- [x] I can simulate the issue easily

## Description
<!-- If relevant, attach screenshots, and how to reproduce -->

I wanted to test the "beta" version of desktop.

```
➜  ~ sudo snap install rocketchat-desktop --beta
rocketchat-desktop (beta) 3.0.0-develop par Rocket.Chat (rocketchat✓) installé
➜  ~ rocketchat-desktop                                                  
A JavaScript error occurred in the main process
Uncaught Exception:
Error: Cannot find module 'tslib'
Require stack:
- /snap/rocketchat-desktop/180/resources/app.asar/app/main.js
- 
    at Module._resolveFilename (internal/modules/cjs/loader.js:961:15)
    at Function.o._resolveFilename (electron/js2c/browser_init.js:257:919)
    at Module._load (internal/modules/cjs/loader.js:844:27)
    at Function.Module._load (electron/js2c/asar.js:769:28)
    at Module.require (internal/modules/cjs/loader.js:1023:19)
    at require (internal/modules/cjs/helpers.js:77:18)
    at Object.<anonymous> (/snap/rocketchat-desktop/180/resources/app.asar/app/main.js:3:13)
    at Module._compile (internal/modules/cjs/loader.js:1145:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1166:10)
    at Module.load (internal/modules/cjs/loader.js:981:32)
Gtk-Message: 11:56:54.730: Failed to load module "appmenu-gtk-module"
```

I have installed : 

`➜  ~ sudo apt install --reinstall appmenu-gtk2-module appmenu-gtk3-module
`

But no luck. Maybe the real problem is 

`Error: Cannot find module 'tslib'
`

### Current Behavior
<!-- What actually happens? -->

It doesn't start.

### Expected Behavior
<!-- What do you think should happen? -->

It should start :)

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,10 +1,7 @@
 language: node_js
 node_js: 12
 os: linux
-cache:
-  yarn: true
-  directories:
-    - node_modules
+cache: yarn
 
 jobs:
   include:
@@ -17,14 +14,12 @@ jobs:
     deploy:
       - provider: snap
         cleanup: false
-        skip_cleanup: true
         snap: dist/*.snap
         channel: beta
         on:
           branch: develop
       - provider: snap
         cleanup: false
-        skip_cleanup: true
         snap: dist/*.snap
         channel: stable
         on:
@@ -44,10 +39,29 @@ notifications:
     on_success: never
     on_failure: change
 
-install:
-  - yarn --check-files
-  - yarn test
-
 script:
-  - if [[ "$TRAVIS_PULL_REQUEST" != "false" ]]; then export CSC_IDENTITY_AUTO_DISCOVERY=false; fi
-  - NODE_ENV=production yarn electron-builder --publish onTagOrDraft --x64
+  - yarn --check-files --production=false
+  - export NODE_ENV=production
+  - yarn build
+  - yarn test
+  - |
+      if [[ "$TRAVIS_PULL_REQUEST" != "false" ]]
+      then
+        export CSC_IDENTITY_AUTO_DISCOVERY=false
+      fi
+  - yarn electron-builder --publish onTagOrDraft --x64
+  - |
+      if [[ "$TRAVIS_OS_NAME" == "osx" ]]
+      then
+        ./dist/mac/Rocket.Chat.app/Contents/MacOS/Rocket.Chat 1>output.log 2>error.log &
+        sleep 30
+        kill %1
+      fi
+      if [[ "$TRAVIS_OS_NAME" == "linux" ]]
+      then
+        ./dist/linux-unpacked/rocketchat-desktop 1>output.log 2>error.log &
+        sleep 30
+        kill %1
+      fi
+      cat output.log
+      cat error.log

--- a/appveyor.yml
+++ b/appveyor.yml
@@ -8,7 +8,7 @@ shallow_clone: true
 clone_depth: 1
 cache:
   - "%LOCALAPPDATA%/Yarn"
-  - node_modules -> package.json
+  - node_modules -> yarn.lock
 
 branches:
   only:
@@ -22,7 +22,6 @@ branches:
 
 install:
   - ps: Install-Product node $env:nodejs_version $env:platform
-  - yarn
 
 build_script:
   - ps: |
@@ -33,9 +32,17 @@ build_script:
         [Convert]::FromBase64String($env:CSC_LINK) | Set-Content ($tmp.FullName + ".p12") -Encoding Byte
         $env:CSC_LINK = ($tmp.FullName + ".p12")
       }
-  - yarn test
+  - yarn --check-files --production=false
   - set NODE_ENV=production
+  - yarn build
+  - yarn test
   - yarn electron-builder --publish onTagOrDraft --x64 --ia32
+  - ps: |
+      $p = Start-Process -FilePath ".\dist\win-unpacked\Rocket.Chat.exe" -PassThru -RedirectStandardOutput .\output.log -RedirectStandardError .\error.log
+      Start-Sleep -s 30
+      $p.Kill()
+      Get-Content -Path .\output.log
+      Get-Content -Path .\error.log
 
 # on_finish:
 #  - ps: $blockRdp = $true; iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/appveyor/ci/master/scripts/enable-rdp.ps1'))

--- a/package.json
+++ b/package.json
@@ -20,7 +20,7 @@
   },
   "main": "app/main.js",
   "scripts": {
-    "postinstall": "run-s install-app-deps clean build",
+    "postinstall": "run-s install-app-deps clean",
     "start": "run-s clean build:watch",
     "clean": "rimraf app dist",
     "build": "rollup -c",

--- a/package.json
+++ b/package.json
@@ -59,7 +59,8 @@
     "redux": "^4.0.5",
     "reselect": "^4.0.0",
     "rimraf": "^3.0.2",
-    "semver": "^7.3.2"
+    "semver": "^7.3.2",
+    "tslib": "^2.0.1"
   },
   "devDependencies": {
     "@babel/core": "^7.11.4",
@@ -108,7 +109,6 @@
     "rollup": "^2.26.6",
     "rollup-plugin-copy": "^3.3.0",
     "ts-jest": "^26.3.0",
-    "tslib": "^2.0.1",
     "typescript": "^4.0.2",
     "xvfb-maybe": "^0.2.1"
   },

